topic-operator/pom.xml

```diff
@@ -17,6 +17,10 @@
         </license>
     </licenses>
 
+    <properties>
+        <mockito.version>4.4.0</mockito.version>
+    </properties>
+
```

Comment:
Overrode Mockito version here to pick up better API

Reply:
Any reason not to bump it globally?

Reply:
Nope, have done so now :)